openhtf/plugs/usb/adb_protocol.py

```diff
@@ -154,7 +154,7 @@ def __init__(self, destination, transport):
     self._transport = transport
 
   def is_closed(self):
-    """Return True iff the stream is closed."""
+    """Return True if the stream is closed."""
```

Comment:
I think this was intentional, but in my opinion “if” is sufficient.

Reply:
Yeah, I wasn't sure especially as the number of 'f's just kept going up in subsequent comments. :-)